### DISCUSSION TITLE
Getting text with timecode

### DISCUSSION THREAD

**Discussion by Amanb1145** [openai/whisper#2184] [Q&A]
Hey,
Using this model for converting audio to text using this code-
```
`import whisper

model = whisper.load_model("base")

# load audio and pad/trim it to fit 30 seconds
audio = whisper.load_audio("audio.mp3")
audio = whisper.pad_or_trim(audio)

# make log-Mel spectrogram and move to the same device as the model
mel = whisper.log_mel_spectrogram(audio).to(model.device)

# detect the spoken language
_, probs = model.detect_language(mel)
print(f"Detected language: {max(probs, key=probs.get)}")

# decode the audio
options = whisper.DecodingOptions()
result = whisper.decode(model, mel, options)

# print the recognized text
print(result.text)`
```

but I also want to print the timecodes for the text.
Please let me know if this is possible.
Thanks

**Comment by phineas-pta** [comment]
use the `transcribe()` method

**Reply by station72** [reply]
Hello! Is it possible to get accurate timestamps for different words? Now I am getting approximated to seconds `"start": 100.0, "end": 102.0`, which is unsuitable for my case.

**Reply by ryanheise** [reply]
Yes, search this discussion board for word timestamps, and also use the help option described on the readme page to see the exact syntax of the word timestamps option.
